RELEASENOTES.docu

```diff
@@ -1771,4 +1771,8 @@ extern typedef struct { } my_type_t;</pre> </add-note></build-id>
       m(conf_object_t)</tt> for function pointers is only permitted
       when this feature is enabled. The feature will be disabled in
       Simics API versions > 7.</add-note></build-id>
+  <build-id _6="next" _7="next"><add-note>Fixed a bug in
+      the <tt>dead_dml_methods</tt> library, which caused it to never
+      detect methods as dead if they occur directly after
+      a <tt>footer</tt> block.</add-note></build-id>
 </rn>
```

Comment:
We want a releasenote for the provisional mechanism, documenting its addition (because yes, it deserves it, in the same way `--compat` deserved it.) At the very least, we want it at the same time we offer documentation for provisionals, if we don't do that right away. Seperately we should have a releasenote for `explicit_param_decls`. (These both belong to 1.4's releasenotes.)

Reply:
A release note will come with its documentation.

Reply:
A release note has been written.